docs/install/index.md

```diff
@@ -8,13 +8,35 @@ has_toc: false
 
 # Install GrimoireLab
 
-* pip
-* docker
-* docker-compose
+GrimoireLab offers three options for installation:
 
-## Define analysis scope:
+* [Docker Compose](./docker-compose)
+  - Use `docker-compose` if you want separate containers for the database.
+* [Docker](./docker)
+  - Use `docker` if you want to work with one self-contained container that includes everything.
+* [PIP](./pip)
+  - Use `pip` if you want to use GrimoireLab components individually.
 
-### Intro to projects.json
-### Intro to setup.cfg
+Regardless of which way you install GrimoireLab, you will need to configure it as described below.
 
-## Deploy and run GrimoireLab
\ No newline at end of file
+## Define analysis scope
+
+### Tell GrimoireLab what projects to collect data from
+
+GrimoireLab needs to know what projects to collect data from. 
+The default [projects.json](https://github.com/chaoss/grimoirelab/blob/master/default-grimoirelab-settings/projects.json) file is configured to collect metrics about the GrimoireLab git repository. 
+
+To start collecting data about the projects that matter to you, you need to define projects and data endpoints in the project.json file.
+The [GrimoireLab tutorial section on the projects file](https://chaoss.github.io/grimoirelab-tutorial/sirmordred/projects.html) explains how the file needs to be built.
```

Comment:
As general rule (when possible), it's probably better to point to the doc of the corresponding GitHub repo, in this case: https://github.com/chaoss/grimoirelab-sirmordred#projectsjson

Reply:
Good point! I'll make the change.